File: core/src/main/java/io/nirahtech/erp/core/PhoneNumber.java
package io.nirahtech.erp.core;

public final class PhoneNumber {
    private final int countryCode;
    private final int number;

    public PhoneNumber(int countryCode, int number) {
        this.countryCode = countryCode;
        this.number = number;
    }

    public int getCountryCode() {
        return countryCode;
    }

    public int getNumber() {
        return number;
    }

}

File: webapp/src/main/react/src/data/PhoneNumber.ts
class PhoneNumber {
    private readonly countryCode: number;
    private readonly number: number;
  
    constructor(countryCode: number, number: number) {
      this.countryCode = countryCode;
      this.number = number;
    }
  
    getCountryCode(): number {
      return this.countryCode;
    }
  
    getNumber(): number {
      return this.number;
    }

    toString(): string {
      return String("+".concat(String(this.countryCode)).concat(String(this.number)));
    }
  }

  export default PhoneNumber;